src/utils.ts

```diff
@@ -89,9 +89,11 @@ export const getPlaywrightInstance = async (
     const playwright = require('playwright')
     if (selectors) {
       await Promise.all(
-        selectors.map(({ name, script }) =>
-          playwright.selectors.register(name, script),
-        ),
+        selectors.map(({ name, script }) => {
+          if (!playwright.selectors._engines.get(name)) {
```

Comment:
seems like a weird workaround to access a private field. hmm

Reply:
Of cause this is true, but for now I can't see any others solutions to fix it. https://github.com/playwright-community/jest-playwright/issues/99#issuecomment-609698736